### DISCUSSION TITLE
If a theme is registered, loop through defined "dataColors"

### DISCUSSION THREAD

**Discussion by cbaragao** [NatVanG/PBIXInspector#14] [Ideas]
It appears that if a theme is registered in report.json, the theme will be saved in StaticResources/RegisteredResources. Looking at a theme file I have it appears the colors are defined by position. Possibly loop through and find conflicting theme colors?

**Comment by NatVanG** [comment]
Thanks for the idea @cbaragao! Could you please expand on what you mean by conflicting theme colour?

**Reply by cbaragao** [reply]
This would probably be a heavy lift, but the idea was to check theme colors against each other. Example, if a background color is dark on a visual and the font color is dark as well and there is not enough contrast, this would be hard to read.

Ideally the things people are using to generate custom themes would check for this, but a check would be helpful.
